tests/utils/test.py

```diff
@@ -30,6 +31,10 @@ def pprint_html(response, **selectors):
     print("\n\n".join([elt.prettify() for elt in parser.find_all(**selectors)]))
 
 
+def remove_static_hash(content):
+    return re.sub(r"\.[\da-f]{12}\.svg", ".svg", content)
```

Comment:
```suggestion
    return re.sub(r"\.[\da-f]{12}\.svg$", ".svg", content)
```

Reply:
Si on fait ça on ne peux plus l'utiliser dans `test_includes.py`, mais éventuellement on peux au moins faire ça pour border un peu :
```suggestion
    return re.sub(r"\.[\da-f]{12}\.svg\b", ".svg", content)
```
Ceci devrais aussi fonctionner :
```suggestion
    return re.sub(r"""\.[\da-f]{12}\.svg(?:$|")""", ".svg", content)
```
Mais je pense qu'on va complexifier la chose pour peu de gain et que les copains pourraient nous en vouloir en tombant dessus :sweat_smile:.

Reply:
La suggestion a marché chez moi, les tests sont passés. :thinking:

Reply:
```
(.venv) ~/dev/itou3 (git)-[rsebille/ci-collectstatic] % pytest tests/www/approvals_views/test_includes.py --create-db
======================================================================================= test session starts ========================================================================================
platform linux -- Python 3.11.8, pytest-7.4.4, pluggy-1.2.0
django: version: 5.0.3, settings: config.settings.test (from ini)
Using --randomly-seed=362368215
rootdir: /home/freitafr/dev/itou3
configfile: pytest.ini
plugins: syrupy-4.6.1, Faker-19.1.0, anyio-3.7.1, django-4.8.0, xdist-3.5.0, randomly-3.15.0, subtests-0.12.1, mock-3.14.0, respx-0.21.1
collected 6 items

tests/www/approvals_views/test_includes.py ......                                                                                                                                            [100%]

------------------------------------------------------------------------------------- snapshot report summary --------------------------------------------------------------------------------------
6 snapshots passed.
======================================================================================== 6 passed in 9.10s =========================================================================================
pytest tests/www/approvals_views/test_includes.py --create-db  8,62s user 0,58s system 81% cpu 11,329 total
(.venv) ~/dev/itou3 (git)-[rsebille/ci-collectstatic] % git diff

Δ tests/utils/test.py
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────

──────────────────────────────────────────────┐
• 32: def pprint_html(response, **selectors): │
──────────────────────────────────────────────┘


 def remove_static_hash(content):
-    return re.sub(r"\.[\da-f]{12}\.svg\b", ".svg", content)
+    return re.sub(r"\.[\da-f]{12}\.svg$", ".svg", content)


 def parse_response_to_soup(response, selector=None, no_html_body=False, replace_in_attr=None):
```

Reply:
Évidemment. Parce que j’ai pas `CI=True`.